app/AcquireIntersonArrayRF.cxx

```diff
@@ -217,10 +182,18 @@ int main( int argc, char * argv[] )
   Sleep( 100 ); // "time to stop"
   container.DisposeScan();
 
-  typedef itk::ImageFileWriter< ImageType > WriterType;
+  //
+  typedef itk::Image<short> WriterImageType;
+  typedef itk::CastImageFilter<ImageType, WriterImageType> CasterType;
```

Comment:
The conversion is necessary.  The C# dlls use unsigned short for the internal buffer, and the wrapper code must match that.  However, the data itself is actually signed, and so it needs to be casted to short to view/write it.  I could move the conversion to be inside the wrapper class somewhere?